### PR TITLE
Move pehapkari related czech posts from TomasVotruba.cz

### PR DESCRIPTION
**Pull request by TomasVotruba**
I'll be migrating my personal site to English and there are some Pehápkaři related posts, before this blog existed.

The reason to move them here:

1. I don't want to translate them to English, that would not make sense, since it's local community 
2. I want to keep them alive, just for community history and nostalgy, when we are old :older_man:  :older_woman: 
3. This would be the right place to add them, if this blog would exist in time of their writing

I will redirect old posts here after this is merged.

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Merging for now, since it's formality and blocker for other PR

File: source/_posts/2016/2016-09-26-symfony-komunita-roste-na-pehapkare-a-otevira-se-vsem-php-programatorum.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+Vítáme tak všechny, kteří **jakkoliv dělají s PHP**, v České republice a na Slovensku.
+
+## Čeho se to týká
+
+- Přejmenovali jsme Twitter na **[@pehapkari](http://twitter.com/pehapkari)**
+- Twitter hashtag na [#pehapkari](http://twitter.com/#pehapkari)
+- A Facebook stránku na **[Péhápkaři](https://www.facebook.com/pehapkari)**
+- Symfony srazy budou PHP srazy
+
+Zbyte se dozvíš na webu *pehapkari.cz*, který bude do konce září.
```
Comment (enumag) [inline source/_posts/2016/2016-09-26-symfony-komunita-roste-na-pehapkare-a-otevira-se-vsem-php-programatorum.md]: Zbytek

Comment (TomasVotruba) [inline source/_posts/2016/2016-09-26-symfony-komunita-roste-na-pehapkare-a-otevira-se-vsem-php-programatorum.md]: PR?

Comment (enumag) [inline source/_posts/2016/2016-09-26-symfony-komunita-roste-na-pehapkare-a-otevira-se-vsem-php-programatorum.md]: done

Comment (TomasVotruba) [reply source/_posts/2016/2016-09-26-symfony-komunita-roste-na-pehapkare-a-otevira-se-vsem-php-programatorum.md]: :+1:

**Pull request merged**